src/lib/LUA/tx_devLUA.cpp

```diff
@@ -336,18 +336,23 @@ static void luadevGeneratePowerOpts()
 static void luahandWifiBle(struct luaPropertiesCommon *item, uint8_t arg)
 {
   struct luaItem_command *cmd = (struct luaItem_command *)item;
+  std::function<void()> setTargetState;
   connectionState_e targetState;
   const char *textConfirm;
   const char *textRunning;
   if ((void *)item == (void *)&luaWebUpdate)
   {
-    targetState = wifiUpdate;
+    setTargetState = []() {
```

Comment:
`setTargetState = &setWifiUpdateMode`, no need for a function that just calls a function with the same parameters.

Reply:
True